frontend/src/components/auth/Signup/TermsConditions.tsx

```diff
@@ -0,0 +1,163 @@
+import { ArrowBackIcon, CloseIcon } from "@chakra-ui/icons";
```

Comment:
looking at the figma flow, it seems like the terms and conditions are _after_ the user verifies their email
in which case we wouldn't have all the information in the formData
Can you confirm with design that the flow you have right now is the desired one?

Reply:
confirmed with design that the flow is correct! (the code of conduct comes before the verification email is sent) -- flow is also reflected in figma